litellm/tests/test_langtrace.py

```diff
@@ -0,0 +1,33 @@
+import os
+import sys
+import time
+
+import pytest
+from opentelemetry.sdk.trace.export.in_memory_span_exporter import InMemorySpanExporter
+from langtrace_python_sdk import langtrace
+
+import litellm
+
+sys.path.insert(0, os.path.abspath("../.."))
+
+
+@pytest.fixture()
+def exporter():
+    exporter = InMemorySpanExporter()
+    langtrace.init(batch=False, custom_remote_exporter=exporter)
+    litellm.success_callback = ["langtrace"]
+    litellm.set_verbose = True
+
+    return exporter
+
+
+@pytest.mark.parametrize("model", ["claude-2.1", "gpt-3.5-turbo"])
+def test_langtrace_logging(exporter, model):
+    litellm.completion(
```

Comment:
i just ran this test. Nothing gets logged to langtrace.
<img width="1695" alt="Screenshot 2024-10-12 at 4 12 16 PM" src="https://github.com/user-attachments/assets/9f83cb80-1af2-4e94-a744-f61200a21d23">